.github/workflows/workflow.yaml

```
@@ -41,7 +41,7 @@ jobs:
         if: ${{ runner.os == 'Linux' }}
         run: |
           sudo apt-get update
-          sudo apt-get install -y libgl-dev ${{ matrix.platform.apt-packages }}
+          sudo apt-get install -y libgl-dev libxext-dev ${{ matrix.platform.apt-packages }}
```

https://github.com/dethrace-labs/dethrace/pull/431 fixes the underlying issue by using sdl2 from git.

```suggestion
          sudo apt-get install -y libgl-dev ${{ matrix.platform.apt-packages }}
```